front/src/app/_components/account-overlay/account-overlay.component.html

```diff
@@ -27,9 +27,9 @@
 
 </ng-container>
 
-<ng-container *ngIf="!(isLoggedIn$|async)">
-  <div class="flex-row info-container px-16">
-    <mat-icon>info</mat-icon>
+<ng-container *ngIf="!(isLoggedIn$|async)">    
```

Comment:
```suggestion
<ng-container *ngIf="!(isLoggedIn$|async)">
```

Reply:
Angular changed and Material changed even more. The legacies you see is only related to Material. We can discuss if we want to migrate this or not but I think the future of the geoshop will be in the geoportal and it's not worth investing efforts on migrate material. What do you think? At least, everything is up to date and under maintenance.